doc/sphinxman/source/glossary_psivariables.rst

```diff
@@ -279,6 +284,11 @@ PSI Variables by Alpha
    The dipole polarizability [au] calculated at the CCSD level
    for a given (x) wavelength, (x) rounded to nearest integer.
 
+.. psivar:: CCSD QUADRUPOLE POLARIZABILITY @ xNM
+
+   The quadrupole polarizability [au] calculated at the CCSD level
```

Comment:
```suggestion
   The quadrupole polarizability in atomic units [(e^2 a0^2)/E_h] calculated at the CCSD level
```
mercy on the user :-)

Reply:
I had better update the dipole polarizability docstring as well, then...